recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/kessel/KesselService.java

```diff
@@ -21,6 +21,8 @@ public class KesselService {
 
     static final String SUBJECT_TYPE_USER = "user";
```

Comment:
This one needs to be `principal`: https://github.com/RedHatInsights/rbac-config/blob/97d7501a55cff790dc44162a6ede8412be2528a8/configs/prod/schemas/schema.zed#L13